### PR TITLE
Boku price row-refs are 0-based (bug 1128012)

### PR DESCRIPTION
**Pull request by kumar303**

### CODE REVIEW

File: lib/boku/tests/test_client.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+from decimal import Decimal
 import time
 import urlparse
```
Comment (andymckay) [inline lib/boku/tests/test_client.py]: i think import before from?

**Comment by andymckay** [comment]
Kinda surprised this happened, I swear we got to testing prices and payments on payments-alt, so I'm confused why it worked there. Or did it ever really work?

File: lib/boku/tests/test_client.py (hunk 1 above)
Comment (kumar303) [inline lib/boku/tests/test_client.py]: oh duh, forgot

**Comment by andymckay** [comment]
r+ wc

**Pull request merged**